src/dialects/postgres/schema/tablecompiler.js

```diff
@@ -90,6 +90,11 @@ TableCompiler_PG.prototype.comment = function(comment) {
   this.pushQuery(`comment on table ${this.tableName()} is '${this.single.comment}'`);
 };
 
+TableCompiler_PG.prototype.check = function(logic) {
+  const name = this.formatter.wrap(`${this.tableNameRaw}_check`);
+  this.pushQuery(`alter table ${this.tableName()} add constraint ${name} check (${logic})`);
```

Comment:
needs to support raw() syntax, or otherwise identifier/value bindings cannot be handled properly across the dialects